src/Adapter/MSTest.TestAdapter/MSTest.TestAdapter.nuspec

```diff
@@ -88,9 +89,12 @@
 
     <!-- net9.0 -->
     <file src="net9.0\build\common\MSTest.TestAdapter.props" target="build\net9.0\MSTest.TestAdapter.props" />
-    <file src="net9.0\build\common\MSTest.TestAdapter.targets" target="build\net9.0\MSTest.TestAdapter.targets" />
+    <file src="net9.0\build\uwp\MSTest.TestAdapter.targets" target="build\net9.0\MSTest.TestAdapter.targets" />
+    <file src="net9.0\Microsoft.VisualStudio.TestPlatform.TestFramework.dll" target="build\net9.0\" />
```

Comment:
Same here, curious why is this needed

Reply:
Reverted to using `net9.0\build\common\MSTest.TestAdapter.targets`